sql-server-2014-alwayson-existing-vnet-and-ad/azuredeploy.json

```diff
@@ -0,0 +1,531 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+
+    "newStorageAccountNamePrefix": {
```

Comment:
Since storage accounts cause so many failures, we're thinking about not asking the user for these and hardcode these unless they are really required. What do you think? see guidance in the follow up comment to previous proposal here https://github.com/Azure/azure-quickstart-templates/issues/944#issuecomment-166189871